jvcl/run/JvAni.pas

```diff
@@ -40,7 +40,8 @@ interface
   {$IFDEF UNITVERSIONING}
   JclUnitVersioning,
   {$ENDIF UNITVERSIONING}
-  Classes, RTLConsts, Windows, Graphics, Controls, ExtCtrls, Dialogs,
+  Types, Classes, RTLConsts, Windows, Graphics, Controls, ExtCtrls, Dialogs,
+  {$IFDEF DELPHI2009_UP} UITypes, {$ENDIF}
```

Comment:
You should really be using `HAS_UNIT_SYSTEM_UITYPES` for the `IFDEF` and `System.UITypes`for the use.

Reply:
OK, i'll fix that.